books/free-programming-books-tr.md

```diff
@@ -39,12 +39,12 @@
 * [Android Dersleri](https://umiitkose.com/android) - Ümit Köse
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
 * [Android Türkçe PDF](http://umiitkose.com/wp-content/uploads/2015/08/AndroidStudio.pdf) - Ümit Köse (PDF)
+* [Android Dersleri](https://bilisim.io/category/mobile/android) - Bilisimio
```

Comment:
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

Reply:
But the resource doesn't seem to be a book.

Reply:
yes you are right but already there are some blog so I added too